pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -160,7 +167,17 @@ let
       + ''
         rm $out/bin/nvim
         touch $out/rplugin.vim
-        makeWrapper ${lib.escapeShellArgs finalMakeWrapperArgs} ${wrapperArgsStr}
+
```

Comment:
should we check for existence of the path first?

Reply:
this wraps the neovim executable: if it didn't exist the build wouldn't work. (i.e., it exists)

Reply:
Ah. I suppose then just the which is relevant